From 1201f19a53e87fbf99018eaef4bbe8d832f114c2 Mon Sep 17 00:00:00 2001
From: Sean Davis <smd.seandavis@gmail.com>
Date: Thu, 13 Sep 2012 01:00:19 +0000
Subject: Applied fix for bug 9270

---
diff --git a/src/parole-player.c b/src/parole-player.c
index 0af6a6f..7a13c99 100644
--- a/src/parole-player.c
+++ b/src/parole-player.c
@@ -1499,7 +1499,7 @@ parole_player_move_fs_window (ParolePlayer *player)
     
     gtk_window_resize (GTK_WINDOW (player->priv->fs_window), 
 		       rect.width, 
-		       player->priv->play_box->allocation.width);
+		       player->priv->play_box->allocation.height);
     
     gtk_window_move (GTK_WINDOW (player->priv->fs_window),
 		     rect.x, 
--
cgit v0.9.0.3
